### DISCUSSION TITLE
Not able to open app on MacOS

### DISCUSSION THREAD

**Discussion by ExpressHermes** [massCodeIO/massCode#43] [Q&A] [answered]
I keep getting this error when trying to open the app
```
“massCode” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

Chrome downloaded this file today at 11:49 AM from masscode.io.
```

I am using macOS Monterey Version 12.2.1

Can someone suggest a solution to this issue?

**Comment by antonreshetov** [comment, marked as answer]
This is the standard security message from macOS. 

1. Go to System Preference
2. Click Security & Privacy
3. Select the Allow button under the Allow apps downloaded from section

Unfortunately, I don't have the ability to buy the Apple Developer Program to have a signature for the app.

**Comment by RohanKambleDev** [comment]
<img width="964" alt="Screenshot 2022-08-18 at 8 56 05 AM" src="https://user-images.githubusercontent.com/968285/185286427-c8acf9ec-7802-42ff-890a-e749c5ec44cb.png">

I am using MacOS Monterey, and I can see only two options under "Allow apps downloaded from", attached is the screenshot.
Hope I am looking at the right place.

**Reply by antonreshetov** [reply]
@rohu2187 Since version 3.2 masOS builds are signed, so there should not be such a problem.

**Reply by RohanKambleDev** [reply]
ok, It worked actually, when I closed the alert popup, I got an option under "Allow apps downloaded from:" saying "do you still want to open massCode" and there you mark it as allow.

Thanks!!
